Makefile

```diff
@@ -19,11 +19,27 @@ tmp/dev_image_id:
 format: prepare
 	${DOCKRUN} bash ./scripts/format.sh
 
-check: prepare
+check: prepare format
 	${DOCKRUN} bash ./scripts/check.sh
 
-todo: prepare
+todo:
 	${DOCKRUN} bash ./scripts/todo.sh
 
-test: prepare format check
-	${DOCKRUN} bash ./scripts/test.sh
+db_seed:
+	docker run --name mysql-db -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -p 3306:3306 -d mysql:5.6
+	docker run --name mongo-db -p 27015:27015 -d mongo:4.2.0
+	docker cp apna_school.sql mysql-db:apna_school.sql
```

Comment:
And this
```
	docker cp apna_school.sql mysql-db:apna_school.sql
	docker exec mysql-db sh -c 'mysql -uroot < apna_school.sql'
```
should be `db_seed`
No need to seed mongo. Because mongo creates keyspaces if it doesn't exist automatically.